worlds/noita/Items.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+from BaseClasses import Item
+
+class NoitaItem(Item):
+    game: str = "Noita"
+
+# 110000 - 110013
+item_table: Dict[str, int] = {
+    "Bad":              110000,
+    "Heart":            110001,
+    "Refresh":          110002,
+    "Potion":           110003,
```

Comment:
Is potion just a random flask? Would it be better to call it "Random Flask" and also include all common element flasks separately?

Reply:
At the moment its just a random flask, but we could switch to named potion elements and put them all in the pool. Pouches should also definately be in here as well.

Reply:
Revisiting this comment, Noita in-game calls it "Potion" so it should keep it as Potion. Everyone I've heard calls it a flask though 🤷🏻 .